File: Computabilidade-e-Complexidade-de-Algoritmos-main/python/18.py
def afd_impar_zeros_uns(s):
    count_0 = s.count('0')
    count_1 = s.count('1')
    return count_0 % 2 != 0 and count_1 % 2 != 0

# Testes
print(afd_impar_zeros_uns("101"))   # True
print(afd_impar_zeros_uns("1100"))  # False

File: Computabilidade-e-Complexidade-de-Algoritmos-main/java/18.java
public class AFDImparZerosUns {
    public static boolean imparZerosUns(String s) {
        int count0 = 0, count1 = 0;
        for (char c : s.toCharArray()) {
            if (c == '0') count0++;
            else if (c == '1') count1++;
        }
        return count0 % 2 != 0 && count1 % 2 != 0;
    }

    public static void main(String[] args) {
        System.out.println(imparZerosUns("101"));   // True
        System.out.println(imparZerosUns("1100"));  // False
    }
}
